### DISCUSSION TITLE
Integração com site Apoia-se

### DISCUSSION THREAD

**Discussion by jhonymiler** [dev-pira/site#148] [Ideas]
Criar integração com o site do apoia-se, onde seria possível validar um membro, inclusive para publicar os artigos da issue #97 
Arrecadar doações de apoiadores e etc

**Comment by murilobeltrame** [comment]
Conta mais sobre essa ideia @jhonymiler ?

**Comment by murilobeltrame** [comment]
@jhonymiler , pegando essa ideia, 
Acho que não tem relação direta com a #97  .... talvez pra isso de pra contar com o que ja temos organizado no Medium https://medium.com/devpira ...
Mas a ideia pareceu legal. Criamos as issues #205  e #206  para manter o track da ideia e implementar.
Se puder colocar seus comentários lá, será muito bemvindo.
